label_sync/main.go

```diff
@@ -758,6 +761,26 @@ func main() {
 	}
 }
 
+func syncWhiteListRepos(config Configuration, onlyRepos string, gc client) {
+	whiteList := strings.Join(config.OnlyRepos, ",")
```

Comment:
we should do the defaulting after the parsing, no need to take structured data and stringify it just to parse it back out

Reply:
Thank you for the review. This was the result of an attempt to avoid any duplication of existing parsing logic. I have made a change to avoid that as well (if the overall change by the PR, i.e. adding an `only` field to configuration, is approved)